### DISCUSSION TITLE
Erron on deployment /bin/sh: rollup: command not found

### DISCUSSION THREAD

**Discussion by netgfx** [vercel/vercel#10122] [Help]
### Problem Description

The deployment fails when it tries to run `rollup -c` script with `/bin/sh: rollup: command not found`

rollup is added as a dependency. Not sure what else does Vercel needs.

**Comment by amyegan** [comment]
Assuming everything works correctly on your local builds, this kind of problem could be caused by a bad lockfile. Does the error go away if you re-generate the lockfile? Or does that cause you to see the same error on local builds?

**Reply by netgfx** [reply]
I don't have local builds, I was building through code-sandbox and deploying on vercel. The code-sandbox is running fine

**Reply by amyegan** [reply]
I want to be sure I understand your workflow correctly. Are you using the CLI to build through code-sandbox and then deploy the prebuilt site code? Or are you pushing the code to a connected GitHub, GitLab, or Bitbucket repo for automatic deployment to Vercel?

**Reply by netgfx** [reply]
I'm coding in codesandbox and pushing the code for deployment through the build-in codesandbox functionality. There is no CLI or local build in the process, its all online through what capabilities codesanbox provides.
From my own investigation it seems that the environment that tries to build the codesanbox code in Vercel doesn't contain the necessary components to run some tasks like rollup or tsc (even if I add them as dependencies on the package.json)
![image (8)](https://github.com/vercel/vercel/assets/864248/5e1ff491-25e9-4bcb-80ce-c56a6b8ba9ba)
